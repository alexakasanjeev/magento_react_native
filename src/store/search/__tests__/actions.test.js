import {
  getSearchProducts
} from '../actions';

describe('search actions', () => {
  test('getSearchProducts()', () => {
    // Setup
    const searchInput = 'Jacket';
    const offset = 0;
    const sortOrder = '';

    // Exercise
    const result = getSearchProducts(searchInput, offset, sortOrder);

    // Verify
    expect(result).toMatchSnapshot();
  });
});
