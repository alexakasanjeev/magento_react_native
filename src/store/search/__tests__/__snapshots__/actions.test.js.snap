// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search actions getSearchProducts() 1`] = `
Object {
  "payload": Object {
    "offset": 0,
    "searchInput": "Jacket",
    "sortOrder": "",
  },
  "type": "MAGENTO_SEARCH_PRODUCTS_REQUEST",
}
`;
